### PR TITLE
create service skeleton

### PR DESCRIPTION
**Pull request by PavelBrest**
### TODO:
- [x] Add GodelTech packages
- [x] Add CQRS module
- [x] Add health check module
- [x] Add DB module
- [x] Add CI pipeline
- [ ] ~~Add Identity module~~
- [ ] ~~Add CD pipeline~~

### CODE REVIEW

File: .github/workflows/dotnet-CI.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.x
+
+    - name: Restore dependencies
+      run: dotnet restore
+
+    - name: Build
+      run: dotnet build --no-restore --configuration Release
```
Comment (f1nal3) [inline .github/workflows/dotnet-CI.yml:25]: ha-ha newline goes brrrrrrr

File: PavValHackathon.Web.sln [hunk 2]
```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (f1nal3) [inline PavValHackathon.Web.sln:2]: Solutions huehuehue

File: src/PavValHackathon.Web.API/Initializers/ServiceDBInitializer.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        public ServiceDBInitializer(IConfiguration configuration) : base(configuration)
+        {
+        }
+
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            var migrationsAssembly = typeof(Startup).GetTypeInfo().Assembly.GetName().Name;
+            var connectionString = Configuration.GetConnectionString("write");
+
+            services.AddDbContext<DataContext>(o => o.UseSqlServer(connectionString,
```
Comment (f1nal3) [inline src/PavValHackathon.Web.API/Initializers/ServiceDBInitializer.cs:24]: dafaq is => ????????

File: src/PavValHackathon.Web.API/Migrations/20210424103829_InitialCreate.Designer.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace PavValHackathon.Web.API.Migrations
+{
+    [DbContext(typeof(DataContext))]
+    [Migration("20210424103829_InitialCreate")]
+    partial class InitialCreate
+    {
+        protected override void BuildTargetModel(ModelBuilder modelBuilder)
+        {
+#pragma warning disable 612, 618
```
Comment (f1nal3) [inline src/PavValHackathon.Web.API/Migrations/20210424103829_InitialCreate.Designer.cs:17]: lewl 1st commit disabling warnings

Comment (f1nal3) [reply src/PavValHackathon.Web.API/Migrations/20210424103829_InitialCreate.Designer.cs:17]: in auto generated code ROFL

File: src/PavValHackathon.Web.API/v1/Contracts/WeatherForecastContract.cs [hunk 5]
```diff
@@ -0,0 +1,17 @@
+using System;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace PavValHackathon.Web.API.v1.Contracts
+{
+    [SwaggerDiscriminator("WeatherForecast")]
+    public record WeatherForecastContract
```
Comment (f1nal3) [inline src/PavValHackathon.Web.API/v1/Contracts/WeatherForecastContract.cs:7]: nigga what

Comment (f1nal3) [reply src/PavValHackathon.Web.API/v1/Contracts/WeatherForecastContract.cs:7]: weather? wtf

**Review by f1nal3 — APPROVED** [review]
noice

File: src/PavValHackathon.Web.API/v1/Contracts/WeatherForecastContract.cs (hunk 5 above)
Comment (pNovikau) [reply src/PavValHackathon.Web.API/v1/Contracts/WeatherForecastContract.cs:7]: ahh, ignore the above. It was generated by a default template.

File: src/PavValHackathon.Web.API/Migrations/20210424103829_InitialCreate.Designer.cs (hunk 4 above)
Comment (pNovikau) [reply src/PavValHackathon.Web.API/Migrations/20210424103829_InitialCreate.Designer.cs:17]: honestly, I have no clue why a code generator ignores an `obsolete` warning.

File: src/PavValHackathon.Web.API/Initializers/ServiceDBInitializer.cs (hunk 3 above)
Comment (pNovikau) [reply src/PavValHackathon.Web.API/Initializers/ServiceDBInitializer.cs:24]: lambda xD
`() => {...};` == `[] () {...};`

**Pull request merged**